### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by SeR4h**
@dennisja Please review these files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.env
+node_modules/
```
Comment (dennisja) [inline .gitignore]: Add a new line at the end of this file

File: .env-example [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#this project uses the apiKey environment variable
```
Comment (dennisja) [inline .env-example]: These comments are very many just add like one comment which explains the purpose of the file and explain each environment variable with a comment on top.
e.g
```sh
# A sample file containing some of the environment variables you will need to set the application

# The API key is given after registering to use the News API
API_KEY=YOUR_NEWS_API_KEY
```

File: README.md [hunk 3]
```diff
@@ -1 +1,21 @@
-# andelaweek4challenge
\ No newline at end of file
+# andelaweek4challenge
+Consuming a news api(node.js)
```
Comment (dennisja) [inline README.md]: These instructions are wrong. Can you look at [this ](https://github.com/dennisja/some-tweets/blob/master/README.md) or [this](https://github.com/dennisja/react-tips-and-patterns/blob/master/README.md) and see how you can improve your readme.

You are going to use Markdown to write the Readme. Read about how to format your readme [here](https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax)

**Review by dennisja — CHANGES_REQUESTED** [review]

**Comment by SeR4h** [comment]
@dennisja please review [additionalFiles] and [readme] for changes

File: README.md [hunk 4]
```diff
@@ -1 +1,27 @@
...
+- npm
+# Setting up the project
+1. Clone the repository and move to the root of the project
+```
+git clone https://github.com/SeR4h/andelaweek4challenge.git
+cd andelaweek4challenge
+```
+2. Install dependencies
+```
+npm install request --save
```
Comment (dennisja) [inline README.md]: Hey @SeR4h 
Well done on this.
All a person needs to do when they clone your project to install your dependencies is to just run 
```sh
npm install
```
No need to install them seperately

**Review by dennisja — APPROVED** [review]
Change the instructions for installing dependencies and merge

**Pull request merged**